src/apis/auth.ts

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+export interface LoginPayload {
```

Comment:
```suggestion
export interface ILoginPayload {
```
앞으로는 interface 이름은 타입과 구분할 수 있도록 I를 앞에 붙여주시고, 추가로 다른곳에서도 쓸 수 있으면 꼭 `src/types` 폴더에 따로 빼주시길 부탁드립니다!

Reply:
넵 알겠습니다! 혹시 이것도 types 폴더로 옮겨야 할까요??

Reply:
이 인터페이스를 그대로 `LoginForm`에서도 쓰면 좋을 것 같네요
types 폴더로 옮기시고, `LoginForm`에서 적용해주세요!